data-collection/src/test/java/refactoringml/ProcessMetricTest.java

```diff
@@ -10,7 +10,8 @@
 
 	@Test
 	public void authorOwnership() {
-		ProcessMetric pm = new ProcessMetric("a.Java", "123", Calendar.getInstance());
+		//TODO: What commit message should I use here?
```

Comment:
Good point! I think the commit message should be of the commit that introduces the refactoring. We do not care much about the last commit of the class (which was probably related to a normal evolution)